lib/v2/kloteneranzeiger/radar.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+    'kloteneranzeiger.ch': {
+        _name: 'Klotener Anzeiger',
+        '.': [
+            {
+                title: 'Klotener Anzeiger New',
```

Comment:
https://docs.rsshub.app/joinus/new-radar#title

> Do not repeat the website name (`_name`)

```suggestion
                title: 'News',
```